### ISSUE TITLE
fast-shadows of apple emoji aren't colored

### ISSUE THREAD

**Issue by Gankro** [servo/webrender#1959]
Now that we correctly prevent emoji from being effected by colors, the colors we try to apply to them as fast shadows get ignored.

test case:

```
<span style="text-shadow: 2px 2px 0px red">
    Shadow
    <span style="color: blue">🎉</span>
    <span style="color: rgba(0,0,0,0.7)">🎉</span>
    <span style="color: rgba(0,0,255,0.4)">🎉</span>
    <span style="color: transparent">🎉</span>
</span>
```

screen shot (with blurred version for reference)

<img width="266" alt="screen shot 2017-10-27 at 11 29 31 pm" src="https://user-images.githubusercontent.com/1136864/32130885-c0dc7a2a-bb6e-11e7-9181-733f25d29f79.png">

**Comment by Gankro** [comment]
live version is near the top of https://gankro.github.io/blah/webtests/text.html

**Comment by lsalzman** [comment]
Gecko bug https://bugzilla.mozilla.org/show_bug.cgi?id=1425157

**Comment by glennw** [comment]
Fixed in #2223.
